CONTRIBUTING.md

```diff
@@ -66,15 +66,18 @@ TIP: if you want you can use prefix eg "CSS" or "JS" for branch naming and in yo
 - make sure to use only class and id
 - Try to use ids or combination of selectors so we won't mix our styles
 - If you decide to use classes use prefix of your task which you are working on eg  `footer-link` `nav-search-button`
-- Avoid tag name like main or section or h1 or p this can mix your styles with other styling or you can also use inline styling in your elements
+- Avoid tag name like `main` or `section` or `h1` or `p` this can mix your styles with other styling
+- Avoid Inline Styling
 
 ### HTML
 
+- make sure to install and use [W3C Web Validator](https://marketplace.visualstudio.com/items?itemName=CelianRiboulet.webvalidator) VSCode Extension help you to validate your Html file
 - use Semantic html like  `section` `article` tags for content
 - avoid generic html and use `div` and `span` only for styling purposes
 
 ### Assets

```

Comment:
At the moment there is no assets folder. 
Assets could include images, fonts, icons or other types of files. 
Are we going to have an assets folder on main folder and use subfolders inside?